src/vivarium_sodium_reduction/model_specifications/model_spec.yaml

```diff
@@ -3,37 +3,62 @@ components:
         population:
             - BasePopulation()
             - Mortality()
-        metrics:
-            - DisabilityObserver()
-            - MortalityObserver()
+        disease:
+            - SI('ischemic_heart_disease')
+            - SI('ischemic_stroke')
+#        metrics:
+#            - DisabilityObserver()
+#            - MortalityObserver()
+    vivarium_sodium_reduction.components:
+        - CorrelatedRisk('risk_factor.high_systolic_blood_pressure')
+        - CorrelatedRisk('risk_factor.diet_high_in_sodium')
+        - RiskCorrelation('risk_factor.high_systolic_blood_pressure', 'risk_factor.diet_high_in_sodium', '0.75')
+        - SodiumSBPEffect()
+    vivarium_public_health.risks:
+        - NonLogLinearRiskEffect('risk_factor.high_systolic_blood_pressure',
+                                'cause.ischemic_heart_disease.incidence_rate')
+    #     - NonLogLinearRiskEffect('risk_factor.high_systolic_blood_pressure',
+    #                             'cause.ischemic_stroke.incidence_rate')
+    vivarium_sodium_reduction.components:
```

Comment:
This component needs to be moved into the vivarium_sodium_reduction.components block above because we unfortunately can't have two keys with the same name in our model spec.

Reply:
Oh, annoying. It would be nice if that were allowed, or at least had a clearer error message to indicate that it is not allowed.  Can you add a JIRA ticket to the backlog about it for me, @hussain-jafari ?

Reply:
Done! Ticket at https://jira.ihme.washington.edu/browse/MIC-5256